recipes/zippy/all/conanfile.py

```diff
@@ -0,0 +1,31 @@
+from conans import ConanFile, tools
+import os
+
+class NoWideConan(ConanFile):
+    name = "zippy"
+    description = "A simple C++ wrapper around the \"miniz\" zip library "
+    topics = ("wrapper", "compression", "zip")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/troldal/Zippy"
+    license = "MIT"
+    exports_sources = "patches/*"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("miniz/2.2.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def package(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*.hpp", dst="include", src=os.path.join(self._source_subfolder, "library"))
+
+    def package_id(self):
+        self.info.header_only()
```

Comment:
Although the cmake script does not install/export it, the cmake script creates a `Zippy::Zippy` target.
Perhaps it's nice to also create it?
Using `Zippy::Zippy` looks reasonable + it will make usage of this package easier.
https://github.com/troldal/Zippy/blob/a838de8522f9051df0d1b202473bb6befe648702/library/CMakeLists.txt#L10
Of course, a comment should be added about it being unstable.
@prince-chrismc for his opinion

Reply:
I think that's reasonable... if they ever install targets I'd expect them to be the same